api/envoy/config/route/v3/route_components.proto

```diff
@@ -1742,6 +1763,9 @@ message RateLimit {
 
       // Rate limit on metadata.
       MetaData metadata = 8;
+
+      // Rate limit on a computed expression value.
+      Expression expr = 9;
```

Comment:
Please add a `[#not-implemented-hide:]` comment here.

Reply:
Implemented.